README.md

```diff
@@ -45,20 +45,28 @@ console.log(languages.getNames("en")); // { 'ab': 'Abkhazian', 'aa': 'Afar', [..
 
 ### Supported languages (ISO 639-1)
 
+* `ar`: Arabic (based on ChatGPT)
+* `bg`: Bulgarian (based on ChatGPT)
 * `br`: Breton (based on https://br.wikipedia.org/wiki/Listenn_glok_kodoù_ISO_639-1)
+* `ca`: Catalan (based on ChatGPT)
 * `cs`: Czech (based on https://cs.wikipedia.org/wiki/Seznam_kódů_ISO_639-1)
 * `da`: Danish (based on https://da.wikipedia.org/wiki/ISO_639-1)
 * `de`: German (by native speaker)
+* `el`: Greek (based on ChatGPT)
 * `en`: English (ISO 639-1 standard names)
 * `es`: Spanish (based on https://es.wikipedia.org/wiki/ISO_639-1)
+* `et`: Estonian (based on ChatGPT)
+* `fa`: Persian (based on ChatGPT)
 * `fi`: Finnish (based on https://kotoistus.fi/suositukset/suositukset-kielet-fi-koodi/)
 * `fr`: French (based on https://fr.wikipedia.org/wiki/Liste_des_codes_ISO_639-1)
 * `he`: Hebrew (based on ECMAScript Internationalization API data)
```

Comment:
I see this content is being modified partially, if it's also based on chatGPT, would make sense to update the origin?(as mixed or combined depending on how you updated it)

Reply:
I was referring `he` that is being modified in the PR too. I think only the new ones should be modified, otherwise, the origin data specified for others are being altered and not updated.